crates/usvg/src/parser/converter.rs

```diff
@@ -7,15 +7,15 @@ use std::hash::{Hash, Hasher};
 use std::str::FromStr;
 use std::sync::Arc;
 
-#[cfg(feature = "text")]
-use fontdb::Database;
-use svgtypes::{Length, LengthUnit as Unit, PaintOrderKind, TransformOrigin};
-
 use super::svgtree::{self, AId, EId, FromValue, SvgNode};
 use super::units::{self, convert_length};
 use super::{marker, Error, Options};
 use crate::parser::paint_server::process_paint;
 use crate::*;
+#[cfg(feature = "text")]
```

Comment:
Keep at the old position.

Are you manually moving import or is it IDE/formatter?

Reply:
Must be the formatter. :(